code/game/objects/effects/aliens.dm

```diff
@@ -181,6 +181,7 @@
 	loc = null
 	for (var/obj/structure/alien/weeds/W in range(1,T))
 		W.updateWeedOverlays()
+	linked_node = null

```

Comment:
This Destroy proc sets its own loc to null. That should only be done in /atom/movable/Destroy().